recipes/paprika/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: paprika
+  version: "0.1.0"
+
+source:
+  path: ../..
```

Comment:
You need to put a URL and sha256 here, see [here for an example](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L13-L22).

Reply:
Great, thank you. I misread that line and thought the `sha256` only applied to a PyPI release.

Reply:
No problem!

The version in that URL doesn't match, using a jinja2 variable like the example does will help keep everything consistent. The `{{ version }}` gets replaced with the version defined at the top of the file with `{% set version = "0.0.4" %}`

Reply:
Also you don't need the `path` entry at all.

Reply:
Yes, got it. I wasn't sure if you (i.e., conda-forge) was going to replace the variables or you were just showing how to programmatically write a `meta.yaml`. Will fix in a sec.

Reply:
Hopefully fixed in 6f94ea7

Reply:
Any thoughts on `subprocess.CalledProcessError: Command '['/bin/bash', '-e', '/home/conda/feedstock_root/build_artifacts/paprika_1550788710742/work/conda_build.sh']' returned non-zero exit status 1.`?

Looks like it Azure can't find `pip`? 

`/home/conda/feedstock_root/build_artifacts/paprika_1550788710742/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/python: No module named pip`

Reply:
`pip` should be in the `host` requirements.

Reply:
Thanks again @chrisburr .

Any idea what's up with the Windows builds? https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=15775&view=logs The error is non-obvious to me (looks like something to do with Windows `conda`):
```
  File "C:\Miniconda\lib\site-packages\conda\core\subdir_data.py", line 60, in __call__
    assert channel.subdir
AssertionError
```

However, I should note that we do not internally test on Windows at all, so I'm not confident that our package will work appropriately there in any case.

Also, Travis seems to be perpetually pending.

Reply:
You can ignore Azure win-64, it has known issues (#7715).

The Travis plan conda-forge has can't handle the load unfortunately so the backlog can get pretty bad. It's being worked on so it should improve at some point.

Reply:
Gotcha. So we should wait for Travis to build before trying to merge?